docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -112,8 +115,14 @@ const styles = ({ theme }) => ({
   },
   [`& .${classes.container}`]: {
     paddingTop: 60 + 20,
-    marginBottom: theme.spacing(8),
-    maxWidth: `calc(740px + ${theme.spacing(12)})`,
+    marginBottom: theme.spacing(1),
```

Comment:
```suggestion
    marginBottom: theme.spacing(8),
```

I reckon we should still maintain the `8` value there, otherwise: 

| Before (production)      | After (this PR) |
| ----------- | ----------- |
| <img width="700" alt="Screen Shot 2022-08-02 at 02 06 51" src="https://user-images.githubusercontent.com/67129314/182296173-883ad648-6e25-4251-a6bc-cc173aa3f4cb.png">  | <img width="700" alt="Screen Shot 2022-08-02 at 02 06 43" src="https://user-images.githubusercontent.com/67129314/182296155-558e8140-77c3-4816-84da-b8cacee3f84f.png"> |

More might even be better, though: 

```suggestion
    marginBottom: theme.spacing(12),
```

Reply:
Oops, I didn't do this change on purpose. I have used this to test what was the value of `theme.spacing(1)`. At which point I realized it was 8px and not the expected 10px.